app/src/main/java/ch/epfl/sdp/blindly/recording/AudioLibraryAdapter.kt

```diff
@@ -49,65 +63,84 @@ class AudioLibraryAdapter(var recordList: ArrayList<AudioRecord>,
         val selectButton: Button = view.findViewById(R.id.selectButton)
 
         init {
-            // Define click listener for the ViewHolder's View.
             nameDurationLayout.setOnClickListener(this)
             expandableLayout.visibility = View.GONE
         }
 
         override fun onClick(v: View?) {
-            // Forward the click to the main activity
             if (adapterPosition != RecyclerView.NO_POSITION) {
                 listener.onItemClick(adapterPosition)
             }
         }
     }
 
-    // Expands or collapse the layout for playing the audio file
-    private fun toggleLayout(isExpanded: Boolean, v: View, layoutExpand: RelativeLayout) {
+    /**
+     * Expands or collapse the layout of the audio file, and collapse other ones.
+     *
+     * @param isExpanded if the record is currently expanded in layout
+     * @param view the current view
+     * @param layoutExpand the layout to expand/collapse
+     */
```

Comment:
Are we suppose to add kdocs for private methods ? I think we should add comments but not proper documentation

Reply:
I don't know, maybe we can ask our coaches tomorrow. I think it might be a good idea to do so when the functions aren't trivial, but this one is pretty straightforward so it wouldn't be needed, you're right.

Reply:
Adding comments is always a good idea. I don't know for Kotlin, but the /** in Java is reserved for documentation, or am I confused ? We could add /* or // comments for private functions

Reply:
l'll work on K-doc next week anyway